Aardvark/ARKEmailBugReporter.m

```diff
@@ -335,6 +420,14 @@ - (void)_createBugReportWithTitle:(NSString *)title
                             }
                         }
                         
+                        if (self.viewHierarchyDescription != nil) {
+                            NSData *viewHierarchyData = [self.viewHierarchyDescription dataUsingEncoding:NSUTF8StringEncoding];
+                            if (viewHierarchyData.length) {
```

Comment:
`.length > 0` please. Implicit 0 comparisons are 🤢

Reply:
This and non-const copied from the screenshot section directly above.  Will change both.  :)